rtrlib/rtr/packets.c

```diff
@@ -369,6 +369,11 @@ static bool rtr_pdu_check_size (const struct pdu_header *pdu) {
     //TODO: Check the error msg 0 termination
 
     break;
+  case SERIAL_QUERY:
+  case RESET_QUERY:
+  case RESERVED:
+  default:
```

Comment:
No error required, I implemented checks for SERIAL and RESET QUERY - even though they may never be received. For RESERVED a debug error message is shown. PR is updated.